cmd/ocm-support/account/account_label.go

```diff
@@ -0,0 +1,64 @@
+package account
+
+import (
+	"fmt"
+	"ocm-support-cli/cmd/ocm-support/utils"
+	"ocm-support-cli/pkg/account"
+	"ocm-support-cli/pkg/label"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	v1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	external bool
+}
+
+// CmdCreateAccountLabel represents the create account label command
+var CmdCreateAccountLabel = &cobra.Command{
+	Use:   "accountLabel [accountID] [key] [value]",
+	Short: "Creates a Label to an Account",
+	Long:  "Creates a Label to an Account",
```

Comment:
I think we should keep the text consistent with `cmd/ocm-support/create/cmd.go` and use Assigns instead of Creates in the helpers.

Reply:
Done. Should I make similar changes to readme.md as well?

Reply:
Yes, please.